app/src/main/java/org/kiwix/kiwixmobile/utils/SharedPreferenceUtil.java

```diff
@@ -172,4 +173,10 @@ public void setShowBookmarksCurrentBook(boolean prefShowBookmarksFromCurrentBook
         .putBoolean(PREF_SHOW_BOOKMARKS_CURRENT_BOOK, prefShowBookmarksFromCurrentBook)
         .apply();
   }
+  public void updateLastBookOpened(String title){
```

Comment:
There should be a blank line before the start of the method. You should import the Square's code style and reformat your code using `Ctrl + Shift + L` before committing to avoid such mistakes.